app/Models/Sensor.php

```diff
@@ -54,6 +54,11 @@ public function icon()
         return collect(self::$icons)->get($this->sensor_class, 'delicius');
     }
 
+    public static function getTypes()
+    {
+        return ['Voltage', 'Temperature', 'Fanspeed'];
```

Comment:
@murrant 
you meant this?

Reply:
Yes, but probably need all the types. :)

Reply:
did some homework :D